alien4cloud/alien4cloud_structs.go

```diff
@@ -690,3 +690,59 @@ type CancelExecRequest struct {
 	EnvironmentID string `json:"environmentId"`
 	ExecutionID   string `json:"executionId"`
 }
+
+// CreateUserRequest holds parameters of a requets to create a user
+type CreateUserRequest struct {
```

Comment:
ok, I updated stucts to not directly copy the A4C REST API structs, to avoid duplication of fields.
But not embedding User in CreateUSerRequest, as embedding causes go vet warnings `composite literal uses unkeyed fields` when the struct is used by a client (outside alien4cloud package).